bluesky/tests/test_plans.py

```diff
@@ -171,13 +171,22 @@ def test_reset_wrapper(hw, RE):
     RE(bp.relative_inner_product_scan([], 1,
                                       p.pseudo1, 0, 1,
                                       p.pseudo2, 0, 1))
-    expecte_objs = [p, None, None,
```

Comment:
Heh heh heh. Dollars to donuts this (and the amazingly consistent misspellings that follow) was brought to us by emacs tab-complete.